apps/wallet-dashboard/lib/utils/vesting/getMockedTimelockedStakedObjectsWithDynamicDate.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2024 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { MILLISECONDS_PER_DAY } from '../../constants';
+import { MILLISECONDS_PER_DAY } from '@iota/core';
```

Comment:
This will cause the vesting.spec.ts to import the whole @iota/core package, including JSX files which we don't really want in the tests, that's why the CI is failing

Reply:
If it can't be reverted, then just do something like @iota/core/dasda/constants

Reply:
fixed :pray: